torch/ao/quantization/pt2e/representation/rewrite.py

```diff
@@ -142,19 +180,29 @@ def _reference_dequantize_per_tensor_int8(x_i8, scale, zero_point, quant_min, qu
     # TODO: debug the implementation later when torchdynamo time out issue is resolved
     return ((x_i8.to(torch.float32) - zero_point) * scale).to(dtype=torch.float32)
 
+# boolean flag to indicate if we want to replace literals to be placeholder nodes for pattern and replacement
```

Comment:
I think we need to explain here as well why we need to do this in the first place

Reply:
and maybe add a warning to say this will suddenly fail if the number of literal args or the order of the args are not preserved

Reply:
> and maybe add a warning to say this will suddenly fail if the number of literal args or the order of the args are not preserved

this is probably not possible to do, developers should make sure this is the case I think

Reply:
> I think we need to explain here as well why we need to do this in the first place

this is replaced in future PRs, will add comments later